--- a/build.zig
+++ b/build.zig
@@ -207,7 +207,7 @@
     if (options.with_tracing) {
         try cflags.appendBounded("-DSOKOL_TRACE_HOOKS");
     }
-    if (options.optimize != .Debug) {
+    if (options.optimize != .debug) {
         try cflags.appendBounded("-DNDEBUG");
     }
     switch (backend) {
@@ -341,13 +341,8 @@
     return lib;
 }
 
-// Zig 0.16.0 vs 0.17.0 compatibility helper
 fn addRunFile(b: *Build, p: Build.LazyPath) *Build.Step.Run {
-    if (builtin.zig_version.minor <= 16) {
-        return b.addSystemCommand(&.{p.getPath(b)});
-    } else {
-        return b.addRunFile(p);
-    }
+    return b.addRunFile(p);
 }
 
 //== EMSCRIPTEN INTEGRATION ============================================================================================
@@ -374,11 +369,11 @@
     const emcc_path = emTool(b, options.emsdk, "emcc");
     const emcc = addRunFile(b, emcc_path);
     emcc.setName("emcc"); // hide emcc path
-    if (options.optimize == .Debug) {
+    if (options.optimize == .debug) {
         emcc.addArgs(&.{ "-g", "-Og", "-sSAFE_HEAP=1", "-sSTACK_OVERFLOW_CHECK=1" });
     } else {
         emcc.addArg("-sASSERTIONS=0");
-        if (options.optimize == .ReleaseSmall) {
+        if (options.optimize == .small) {
             emcc.addArg("-Oz");
         } else {
             emcc.addArg("-O3");
--- a/build.zig.zon
+++ b/build.zig.zon
@@ -13,8 +13,7 @@
             .hash = "N-V-__8AAFzfCQCASN6D5b3L8-QbCR9pCMDgNoKp69omdc4V",
         },
         .shdc = .{
-            .url = "git+https://github.com/floooh/sokol-tools-bin#6801e61ab7ea64dd9369ae9ff2f46d20c61fc655",
-            .hash = "sokolshdc-0.1.0-r2KZDj2ESgPeSsOrxWqONyemz4-250cFO8ZhXkEs4DrZ",
+            .path = "../sokolshdc",
         },
     },
 }
